### ISSUE TITLE
temBoard Agent for RHEL 8

### ISSUE THREAD

**Issue by wsescu** [dalibo/temboard-agent#429]
Dear temBoard Team

is there any plan to build a rpm for the temBoard Agent for RHEL 8? Currently, I see only the RHEL7 version on the following page: https://yum.dalibo.org/labs/CentOS7-x86_64/

And the RHEL7 temboard-agent looks not compatible with RHEL8. Maybe related to python 3.

```
$ sudo yum localinstall temboard-agent-6.0-1.el7.noarch.rpm
Updating Subscription Management repositories.
Red Hat Enterprise Linux 8 for x86_64 - BaseOS (RPMs)                                                                                                    95 kB/s | 2.4 kB     00:00
Red Hat Enterprise Linux 8 for x86_64 - Supplementary (RPMs)                                                                                             87 kB/s | 2.1 kB     00:00
Red Hat Enterprise Linux 8 for x86_64 - AppStream (RPMs)                                                                                                117 kB/s | 2.8 kB     00:00
Error:
 Problem: conflicting requests
  - nothing provides python-setuptools needed by temboard-agent-6.0-1.el7.noarch
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
```
Best regards
William

**Comment by bersace** [comment]
Hi @wsescu temBoard 6.0 is not available for RHEL8. As CentOS 8 is not generally available, we should support this soon.

**Comment by wsescu** [comment]
Hello @bersace , as far as I can see, CentOS 8 is available: http://isoredirect.centos.org/centos/8/isos/x86_64/
Do you know approximately, when you will support the temboard-agent on CentOS8/RHEL8?
Thanks a lot.
Best regards, William

**Comment by bersace** [comment]
No idea yet. We have to schedule this.

**Comment by wsescu** [comment]
Would be cool, if you could add this as a feature requests.

**Comment by pgiraud** [comment]
See #470

**Comment by pgiraud** [comment]
Fixed via #470
